packages/icon/src/Icon.stories.tsx

```diff
@@ -76,7 +76,7 @@ Single.argTypes = {
 };
 Single.parameters = { chromatic: { disableSnapshot: true } };
 
-export const AllIcons: StoryFn<IconProps> = (
+export const LiveExample: StoryFn<IconProps> = (
```

Comment:
I think Ln20 in this file should also be updated

Reply:
good catch